statistical_tests/statistical_tests.md

```diff
@@ -76,11 +76,13 @@ Whenever possible, we've linked to tutorials that include explanation of what th
 
 Even when a rich tutorial is available, the information linked here will generally **not** be enough for you to effectively use a statistical technique you're not already familiar with. Before applying a test you don't fully understand, seek out advice from someone experienced in the technique you want to use. We also have [recommendations for further statistical education](#learning-statistics) at the end of this module.
 
-<div class="warning">
-**Important:**
+<div class = "important">
+<b style="color: rgb(var(--color-highlight));">Important note</b><br>
```

Comment:
Hmmm.. I could see it being argued that warning could work here -- we definitely want to draw people's attention to the fact that they should NOT run these tests without understanding the assumptions behind them, etc. In fact, I believe there was previously feedback from a learner who was worried it wasn't emphasized enough. (That might be where we brought in this style, though not sure on that). 

That being said, I'm not against using important note here, either. Just wanted to register this thought (if we're undecided, we can always pull in Ye Olde Module Author, Maine Rose